packages/composite-checkout/src/components/checkout-payment-methods.tsx

```diff
@@ -64,6 +64,7 @@ export default function CheckoutPaymentMethods( {
 						inactiveContent={ paymentMethod.inactiveContent }
 						checked={ true }
 						summary={ true }
+						disabled={ paymentMethod.disabled }
```

Comment:
I think all these `disabled` props were left-over from a previous iteration of this code and can be removed.

Reply:
Removed.